docs/contributing.md

```diff
@@ -1,15 +1,17 @@
 # Developer Contributor Guide
 
-We collaborate on code through git, hosted on GitHub. If you are a software engineer (web/mobile/backend), you should request write access to the codebase.
+We collaborate on code through git, hosted on GitHub. If you are a software engineer (web/mobile/backend), you should request write access to the codebase. If you'd like to contribute regularly, reach out to @aapeliv or @bakeiro to join our Slack channel. This is where our main communication happens. We also do video meetings every week where we discuss priorities, architecture, etc.
```

Comment:
I'm not sure if it's just Aapeli and David who can add someone to Slack so please correct me!

Reply:
I can't 😅

Reply:
but @aapeliv U can add me if U want ;)

Reply:
yeah I sent the invitation to add you, it's just that another person has the final word, but usually it's pretty straighfordward